### PR TITLE
fix the compatiable problem between PY2 and PY3 (issue#20749)

### PR DESCRIPTION
**Pull request by sandyhouse**
Fix issue[#20749](https://github.com/PaddlePaddle/Paddle/issues/20749)
the compatiable problem between PY2 and PY3

### CODE REVIEW

**Review by gavin1332 — APPROVED** [review]
LGTM

File: python/paddle/dataset/sentiment.py [hunk 1]
```diff
@@ -28,6 +28,9 @@
 
 import nltk
 from nltk.corpus import movie_reviews
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
```
Comment (luotao1) [inline python/paddle/dataset/sentiment.py:32]: Why should you add `ssl` here? It is not used in this PR, and if you add this line, we should `pip install ssl` before development. @sandyhouse

Comment (sandyhouse) [reply python/paddle/dataset/sentiment.py:32]: add ssl to fix the issue#20749. I will launch a new pr to add ssl to requirements.txt.

Comment (luotao1) [reply python/paddle/dataset/sentiment.py:32]: You should add `ssl` into `python/requirments.txt` for the usage.

Comment (sandyhouse) [reply python/paddle/dataset/sentiment.py:32]: Right. I have opened the [pr#21669](https://github.com/PaddlePaddle/Paddle/pull/21669) to do that.

**Pull request merged**